### ISSUE TITLE
update installation instruction in README

### ISSUE THREAD

**Issue by konradb3** [jhu-lcsr-forks/rtt_ros_integration#44]
There should be : 
`git clone --recursive git@github.com:jhu-lcsr-forks/orocos_toolchain.git src/orocos/orocos_toolchain`
instead of 
`git clone --recursive git://gitorious.org/orocos-toolchain/orocos_toolchain.git src/orocos/orocos_toolchain`

There are missing dependencies of rtt_ros_integration. There should be at least : 

```
git clone git@github.com:jhu-lcsr-forks/rtt_ros_comm.git
git clone git@github.com:jhu-lcsr-forks/rtt_common_msgs.git
```

**Comment by jbohren** [comment]
> There are missing dependencies of rtt_ros_integration. There should be at least :
> 
> git clone git@github.com:jhu-lcsr-forks/rtt_ros_comm.git
> git clone git@github.com:jhu-lcsr-forks/rtt_common_msgs.git

Yeah, I've been considering whether the packages in rtt_ros_comm should be merged into this repo, and whether rtt_common_msgs should be merged or if anything depending on rtt_common_msgs be pulled out of it.
